lib/OpenQA/WebAPI/Plugin/YAMLRenderer.pm

```diff
@@ -15,8 +15,8 @@
 
 package OpenQA::WebAPI::Plugin::YAMLRenderer;
 use Mojo::Base 'Mojolicious::Plugin';
+use OpenQA::Utils qw(validate_data);
```

Comment:
Maybe use single quotes here for consistency with the line above.

Reply:
done